src/pretix/control/logdisplay.py

```diff
@@ -68,128 +68,212 @@
             new_price=money_filter(Decimal(data['new_price']), event.currency),
         )
-    elif logentry.action_type == 'pretix.event.order.changed.tax_rule':
+
+
+@log_entry_types.new()
+class OrderTaxRuleChanged(OrderChangeLogEntryType):
+    action_type = 'pretix.event.order.changed.tax_rule'
+
+    def display_prefixed(self, event: Event, logentry: LogEntry, data):
         if 'positionid' in data:
-            return text + ' ' + _('Tax rule of position #{posid} changed from {old_rule} '
-                                  'to {new_rule}.').format(
+            return _('Tax rule of position #{posid} changed from {old_rule} to {new_rule}.').format(
                 posid=data.get('positionid', '?'),
                 old_rule=TaxRule.objects.get(pk=data['old_taxrule']) if data['old_taxrule'] else '–',
                 new_rule=TaxRule.objects.get(pk=data['new_taxrule']),
             )
         elif 'fee' in data:
-            return text + ' ' + _('Tax rule of fee #{fee} changed from {old_rule} '
-                                  'to {new_rule}.').format(
+            return _('Tax rule of fee #{fee} changed from {old_rule} to {new_rule}.').format(
                 fee=data.get('fee', '?'),
                 old_rule=TaxRule.objects.get(pk=data['old_taxrule']) if data['old_taxrule'] else '–',
                 new_rule=TaxRule.objects.get(pk=data['new_taxrule']),
             )
-    elif logentry.action_type == 'pretix.event.order.changed.addfee':
-        return text + ' ' + str(_('A fee has been added'))
-    elif logentry.action_type == 'pretix.event.order.changed.feevalue':
-        return text + ' ' + _('A fee was changed from {old_price} to {new_price}.').format(
+
+
+@log_entry_types.new()
+class OrderFeeAdded(OrderChangeLogEntryType):
+    action_type = 'pretix.event.order.changed.addfee'
+    plain = _('A fee has been added')
+
+
+@log_entry_types.new()
+class OrderFeeChanged(OrderChangeLogEntryType):
+    action_type = 'pretix.event.order.changed.feevalue'
+
+    def display_prefixed(self, event: Event, logentry: LogEntry, data):
+        return _('A fee was changed from {old_price} to {new_price}.').format(
             old_price=money_filter(Decimal(data['old_price']), event.currency),
             new_price=money_filter(Decimal(data['new_price']), event.currency),
         )
-    elif logentry.action_type == 'pretix.event.order.changed.cancelfee':
-        return text + ' ' + _('A fee of {old_price} was removed.').format(
+
+
+@log_entry_types.new()
+class OrderFeeRemoved(OrderChangeLogEntryType):
+    action_type = 'pretix.event.order.changed.cancelfee'
+
+    def display_prefixed(self, event: Event, logentry: LogEntry, data):
+        return _('A fee of {old_price} was removed.').format(
             old_price=money_filter(Decimal(data['old_price']), event.currency),
         )
-    elif logentry.action_type == 'pretix.event.order.changed.cancel':
+
+
+@log_entry_types.new()
+class OrderCancelled(OrderChangeLogEntryType):
```

Comment:
```suggestion
class OrderCanceled(OrderChangeLogEntryType):
```
both spellings are correct, but we use the one l everywhere